stock_manual_transfer/models/stock_manual_transfer.py

```diff
@@ -28,9 +28,9 @@ class StockManualTransfer(models.Model):
         states={"draft": [("readonly", False)]},
     )
     route_id = fields.Many2one(
-        "stock.location.route",
+        "stock.route",
         string="Preferred Route",
-        domain=[("manual_transfer_selectable", "=", True)],
+        domain="[('manual_transfer_selectable', '=', True), ('supplied_wh_id', '=', warehouse_id)]",
```

Comment:
Are you sure? Take into account warehouse fields are not mandatory in routes.

Reply:
Yes, I'm experiencing the same issue. 
This feature was requested by Zuly Lira and is already in the production environment. 
When I asked her if this feature was required for all customers, she confirmed that it was a better way to work with the module. This is the origin for that change:
https://git.vauxoo.com/vauxoo/7llantas/-/commit/94a57eca77c58724ef2f3412a0fd921e39d0a420

Here is a snippet of our conversation and her request: 
Zuly Lira, [9/5/24 11:19 A. M.]
Hola @Allan Porras [Vauxoo] recuerdo que hicimos una mejora para que en rutas solo mostrar las relacionadas al almacén seleccionado https://imgur.com/Vq7Mufb y haciendo pruebas ahora en el staging, aparecen todas las rutas https://imgur.com/45Zd5vx
Me ayudas a revisar por favor.
Gracias.

allanpa88, [9/5/24 11:24 A. M.]
Revisando
Igual ese módulo como te había mencionado se va a pasar a addons-vauxoo
o sea, ya no es sólo de Road

allanpa88, [9/5/24 11:25 A. M.]
consideras que es una característica que se deba mantener en el módulo general?

Zuly Lira, [9/5/24 11:19 A. M.]
Hola @Allan Porras [Vauxoo] recuerdo que hicimos una mejora para que en rutas solo mostrar las relacionadas al almacén seleccionado https://imgur.com/Vq7Mufb y haciendo pruebas ahora en el staging, aparecen todas las rutas https://imgur.com/45Zd5vx
Me ayudas a revisar por favor.
Gracias.

allanpa88, [9/5/24 11:23 A. M.]
que extraño, va dejame y reviso

allanpa88, [9/5/24 11:24 A. M.]
Igual ese módulo como te había mencionado se paso a addons-vauxoo

allanpa88, [9/5/24 11:24 A. M.]
o sea, ya no es sólo de Road

allanpa88, [9/5/24 11:25 A. M.]
consideras que es una caraterística que se deba mantener en el módulo general?

Zuly Lira, [9/5/24 11:25 A. M.]
siiii
es mucho mas funcional así...

And it's true, if you select an incorrect route of the entire list, you could get some functional errors difficult to resolve for a non technical or Odoo expert user.